adopt-a-pup/web-app/Jenkinsfile

```diff
@@ -0,0 +1,116 @@
+
+CONTAINER_IMAGE='quay.io/redhattraining/ossm-adopt-a-pup-webapp'
+WEBAPP_DIR='adopt-a-pup/web-app'
+
+
+pipeline {
+    agent {
+        label 'master'
+    }
+
+    environment {
+        HOME = '${WORKSPACE}'
+        NPM_CONFIG_CACHE = '${WORKSPACE}/.npm'
+        CI = 'true'
+    }
+
+    stages {
+        stage('Install') {
+            agent {
+                docker {
+                    image 'node:14'
+                }
+            }
+
+            steps {
+                dir(WEBAPP_DIR) {
```

Comment:
Any other ways to launch the whole pipeline in this directory instead of repeatedly using the `dir` function? 

It tried `customWorkspace` but couldn't make it work.